main/src/main/java/com/sedmelluq/discord/lavaplayer/source/youtube/DefaultYoutubePlaylistLoader.java

```diff
@@ -24,6 +26,8 @@
 public class DefaultYoutubePlaylistLoader implements YoutubePlaylistLoader {
   private static final Logger log = LoggerFactory.getLogger(DefaultYoutubePlaylistLoader.class);
 
+  private static final String REQUEST_URL = "https://www.youtube.com/youtubei/v1/browse?key=AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8";
```

Comment:
there is probably a better way than hardcoding this token, it will probably change over time and break all bots

Reply:
An argument could be had that this should be merged asap (since all bots are currently broken anyways), and making the code more resilient can be done afterwards.

Reply:
I think this key is static, like in YouTube Music, but if someone want scrap this key every time, feel free to do it